packages/lesswrong/components/users/UsersProfileImage.tsx

```diff
@@ -106,12 +106,27 @@ const InitialFallback: FC<{
 });
 
 const UsersProfileImage = ({user, size, fallback="initials", className, classes}: {
-  user: UsersMinimumInfo,
+  user?: UsersMinimumInfo,
   size: number,
   fallback?: ProfileImageFallback,
   className?: string,
   classes: ClassesType,
 }) => {
+  if (!user) {
+    return (
+      <picture className={classes.wrapper}>
+        <div
+          className={classNames(
+            classes.root,
+            classes.loadingPlaceholder,
+            className,
+          )}
+          style={{width: size, height: size}}
+        />
+      </picture>
+    );
+  }
+
```

Comment:
It's not in the Figma. I just added the styles when we created the new user hover over a couple of weeks ago because the images were taking a little while to load (plausibly, it could be faster in the US). The only change here is to also show the same loader when the user is loading too.

Also happy for Agnes to make any changes here!